### ISSUE TITLE
Contributing instructions for GitHub pages documentation

### ISSUE THREAD

**Issue by jtaylor100** [thoughtworks/talisman#351]
**Is your feature request related to a problem? Please describe.**
When I am looking to contribute to the docs on the GitHub pages site, I can't find any instructions on how to do this. I see there is a GitHub pages site on the `gh-pages` branch, but what is the process for contributing change here? Submitting PRs against this branch?

**Describe the solution you'd like**
New section in 'contributing.md' to describe how to make changes to the docs.

**Comment by tinamthomas** [comment]
@jtaylor100 yes it is the gh-pages branch. I can add a section with the process to updating the github pages site.
